src/trust.rs

```diff
@@ -156,6 +156,14 @@ impl TrustOp {
             }
         }
     }
+
+    fn to_string(&self) -> (String, String) {
```

Comment:
lets rename this to something like `to_pair` and make it a function in the module instead of a member

Reply:
How about 'to_string_pair' I'm a big fan of Obvious.

Reply:
to_pair( &TrustOp). Done.